### ISSUE TITLE
«rake aborted!» / «ArgumentError: type» / «lib/site_settings/type_supervisor.rb:112:in `to_rb_value'» on `bundle exec rake db:migrate`

### ISSUE THREAD

**Issue by dmitrii-fediuk** [discourse-pro/df-core#2]
![03](https://user-images.githubusercontent.com/83204/34724143-045977a2-f577-11e7-9801-be82224106ae.png)

```
rake aborted!
ArgumentError: type
lib/site_settings/type_supervisor.rb:112:in `to_rb_value'
lib/site_setting_extension.rb:165:in `block (2 levels) in refresh!'
lib/site_setting_extension.rb:164:in `map'
lib/site_setting_extension.rb:164:in `block in refresh!'
lib/site_setting_extension.rb:161:in `synchronize'
lib/site_setting_extension.rb:161:in `refresh!'
app/models/site_setting.rb:33:in `<class:SiteSetting>'
app/models/site_setting.rb:4:in `<main>'
config/initializers/004-message_bus.rb:84:in `<main>'
config/environment.rb:5:in `<main>'
/home/dfediuk/.rbenv/versions/2.4.2/bin/bundle:23:in `load'
/home/dfediuk/.rbenv/versions/2.4.2/bin/bundle:23:in `<main>'
Tasks: TOP => db:migrate => environment
(See full trace by running task with --trace)
```

**Comment by dmitrii-fediuk** [comment]
https://github.com/discourse/discourse/blob/v1.9.0.beta17/lib/site_settings/type_supervisor.rb#L92-L114

![01](https://user-images.githubusercontent.com/83204/34746879-05e54328-f5c0-11e7-8635-cf622d9dbdbb.png)

**Comment by dmitrii-fediuk** [comment]
I have fixed it by:

```
DELETE FROM site_settings WHERE data_type = 17
```

**Comment by dmitrii-fediuk** [comment]
Fixed in [1.2.7](https://github.com/discourse-pro/df-core/releases/tag/1.2.7).

**Comment by dmitrii-fediuk** [comment]
The same problem: [discourse.pro/t/325](https://discourse.pro/t/325)
